src/api/common/schemas/settings.json

```diff
@@ -12,12 +12,22 @@
     "noFreeze": {"type": "boolean"},
     "globalFreeze": {"type": "boolean"},
     "defaultRipple": {"type": "boolean"},
-    "emailHash": {"$ref": "hash128"},
-    "walletLocator": {"$ref": "hash256"},
-    "walletSize": {"type": "integer"},
+    "emailHash": {
+      "oneOf": [
+        {"type": "null"},
+        {"$ref": "hash128"}
+      ]
+    },
+    "walletLocator": {
+      "oneOf": [
+        {"type": "null"},
+        {"$ref": "hash256"}
+      ]
+    },
+    "walletSize": {"type": ["integer", "null"]},
     "messageKey": {"type": "string"},
     "domain": {"type": "string"},
```

Comment:
I've just took information from [here](https://github.com/ripple/ripple-lib/blob/develop/src/api/common/constants.js#L29) - one with 'defaults' field are clearable. Other from that - don't know.